804.py

```diff
@@ -0,0 +1,11 @@
+from typing import List
+
+class Solution:
+    def uniqueMorseRepresentations(self, words: List[str]) -> int:
+        trans = set()   
+        morse = [".-","-...","-.-.","-..",".","..-.","--.","....","..",".---","-.-",".-..","--","-.","---",".--.","--.-",".-.","...","-","..-","...-",".--","-..-","-.--","--.."]
```

Comment:
It's [recommended](https://www.python.org/dev/peps/pep-0008/#constants) that constants are written in all capital letters. `morse` here is a constant variable in your program.